client/blocks/importer/wordpress/import-everything/pre-migration/migration-credentials-form.tsx

```diff
@@ -29,8 +29,14 @@ interface Props {
 	onChangeProtocol: ( protocol: CredentialsProtocol ) => void;
 }
 
-export const MigrationCredentialsForm: React.FunctionComponent< Props > = ( props ) => {
-	const { sourceSite, targetSite, migrationTrackingProps, startImport } = props;
+export const MigrationCredentialsForm: React.FunctionComponent< Props > = ( {
```

Comment:
I'm getting a Maximum update depth exceeded warning now when opening up the credential form. See the video below:

https://github.com/Automattic/wp-calypso/assets/4074459/63b81f99-a633-40f0-a012-477c0b5123f7